{"name": "[异常]新增机型，有必填项未维护 新增失败", "status": "broken", "statusDetails": {"message": "TypeError: not all arguments converted during string formatting", "trace": "self = <DRPDataMgmt_ModelDatabase.TestAppendProduct object at 0x000002330F720190>, drivers = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"430ada7f1a91a1fabc92781383350d57\")>\n\n    @allure.story(\"新增产品信息\")\n    @allure.title(\"[异常]新增机型，有必填项未维护 新增失败\")\n    @allure.description(\"前往‘产品信息‘菜单，打开新增窗口，不维护大类粗必填项，新增失败\")\n    @allure.severity(\"blocker\")  # blocker\\critical\\normal\\minor\\trivial\n    # @pytest.mark.skip\n    # @pytest.mark.run(order=1)\n    def test_003_004(self, drivers):\n        user = NavPage(drivers)\n        user.click_gotonav(\"DRP数据管理\", \"机型库\")\n        user = DomAssert(drivers)\n        user.assert_url(\"/dataManage/modelLibrary\")\n        user = ModelDatabase(drivers)\n>       user.goto_tab('产品信息')  # 切换到产品信息tab页\n\nproject\\DRP\\test_case\\DRPDataMgmt_ModelDatabase.py:263: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nproject\\DRP\\page_object\\DRPDataMgmt_ModelDatabase.py:18: in goto_tab\n    logging.info(\"前往菜单{}\", format(tab))\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:2097: in info\n    root.info(msg, *args, **kwargs)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:1589: in _log\n    self.handle(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:1599: in handle\n    self.callHandlers(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:1661: in callHandlers\n    hdlr.handle(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:952: in handle\n    self.emit(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\logging.py:331: in emit\n    super().emit(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:1091: in emit\n    self.handleError(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:1083: in emit\n    msg = self.format(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:927: in format\n    return fmt.format(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\logging.py:92: in format\n    return super().format(record)\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:663: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, E:\\工作文件\\DRP_autotest\\新代码\\UIPOMTest\\project\\DRP\\page_object\\DRPDataMgmt_ModelDatabase.py, 18, \"前往菜单{}\">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\nc:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py:367: TypeError"}, "description": "前往‘产品信息‘菜单，打开新增窗口，不维护大类粗必填项，新增失败", "steps": [{"name": "前往菜单", "status": "passed", "parameters": [{"name": "content", "value": "('DRP数据管理', '机型库')"}], "start": 1658220285494, "stop": 1658220288589}, {"name": "值为True值断言", "status": "passed", "parameters": [{"name": "word", "value": "'/dataManage/modelLibrary'"}], "start": 1658220288589, "stop": 1658220288794}, {"name": "前往Tab菜单", "status": "broken", "statusDetails": {"message": "TypeError: not all arguments converted during string formatting\n", "trace": "  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"E:\\工作文件\\DRP_autotest\\新代码\\UIPOMTest\\project\\DRP\\page_object\\DRPDataMgmt_ModelDatabase.py\", line 18, in goto_tab\n    logging.info(\"前往菜单{}\", format(tab))\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 2097, in info\n    root.info(msg, *args, **kwargs)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 1446, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 1589, in _log\n    self.handle(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 1599, in handle\n    self.callHandlers(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 1661, in callHandlers\n    hdlr.handle(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 952, in handle\n    self.emit(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\logging.py\", line 331, in emit\n    super().emit(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 1091, in emit\n    self.handleError(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 1083, in emit\n    msg = self.format(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 927, in format\n    return fmt.format(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\logging.py\", line 92, in format\n    return super().format(record)\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 663, in format\n    record.message = record.getMessage()\n  File \"c:\\users\\jiang.long\\appdata\\local\\programs\\python\\python39\\lib\\logging\\__init__.py\", line 367, in getMessage\n    msg = msg % self.args\n"}, "parameters": [{"name": "tab", "value": "'产品信息'"}], "start": 1658220288794, "stop": 1658220290868}, {"name": "添加失败截图", "status": "passed", "attachments": [{"name": "失败截图", "source": "9d798c58-0da4-4f68-b8e6-17688db04096-attachment.png", "type": "image/png"}], "start": 1658220291345, "stop": 1658220291529}], "attachments": [{"name": "log", "source": "8596db1d-041c-4690-878e-15a9acc0e8d2-attachment.txt", "type": "text/plain"}], "start": 1658220285493, "stop": 1658220290870, "uuid": "7851a618-fc0f-4d02-982a-4ae45345d6bc", "historyId": "294b542997191cd169caf496d5217b82", "testCaseId": "07897d53bdd1e380338455813f4509c4", "fullName": "project.DRP.test_case.DRPDataMgmt_ModelDatabase.TestAppendProduct#test_003_004", "labels": [{"name": "severity", "value": "blocker"}, {"name": "feature", "value": "DRP数据管理-机型库"}, {"name": "story", "value": "新增产品信息"}, {"name": "parentSuite", "value": "project.DRP.test_case"}, {"name": "suite", "value": "DRPDataMgmt_ModelDatabase"}, {"name": "subSuite", "value": "TestAppendProduct"}, {"name": "host", "value": "CQPILONGJIANG"}, {"name": "thread", "value": "18908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "project.DRP.test_case.DRPDataMgmt_ModelDatabase"}]}